### DISCUSSION TITLE
Warning after updating

### DISCUSSION THREAD

**Discussion by elzorrorebelde** [emacs-tree-sitter/elisp-tree-sitter#203] [Q&A] [answered]
after updating and starting Emacs, I get the following warning:

```
Emergency (tree-sitter): Version 0.16.1 of tsc-dyn was already loaded. Please restart Emacs to load the requested version 0.15.2 Disable showing Disable logging
```

![image](https://user-images.githubusercontent.com/1055216/150623791-565c78c3-9fa7-4b56-ac52-5f7481625ed0.png)

How do I fix this?

**Comment by ubolonton** [comment, marked as answer]
This means your `tsc.el` is at 0.15.2, while `tsc-dyn.so` 0.16.1 was somehow loaded already.
It's hard to tell without further details, but you may have better luck asking on Doom's Discord server.

**Comment by elzorrorebelde** [comment]
I just deleted the build-* folders in straight inside Doom Emacs installation, ran doom sync and things went OK. I made the emacs-tree-sitter module for myself some time ago, so I don't know how to prevent this error from happening again

--
Jorge

------- Original Message -------
